conda/dev.yaml

```diff
@@ -14,4 +14,4 @@ dependencies:
   - shellcheck
   - sqlalchemy
   - pip:
-    - pysus
+    - git+https://github.com/luabida/PySUS/@sinan-download
```

Comment:
btw, can we have the official package before merging this PR pls?

Reply:
@luabida is this fixed? This is important.

Reply:
@luabida ?

Reply:
Sorry! Yes, It is fixed.